pctiler/pctiler/reader.py

```diff
@@ -144,3 +174,19 @@ def _reader(
             allowed_exceptions=(TileOutsideBounds, MissingAssets, InvalidAssetName),
             **kwargs,
         )
+
+        te = time.perf_counter()
+
+        logger.info(
+            f"Perf: Mosaic read tile.",
+            extra={
+                "custom_dimensions": {
+                    "duration": f"{te - ts:0.4f}",
+                    "collection": collection,
+                    "zxy": f"{tile_z}/{tile_x}/{tile_y}",
```

Comment:
These are used repeatedly in logs to help identify entries within a request, since we can't add a tracing identifier to the request object in pctiler yet.